### PR TITLE
resolving reviews on env chain pr

### PR DESCRIPTION
**Pull request by souvikmishra**

### CODE REVIEW

File: src/lib/environments/environment.dev.ts [hunk 1]
```diff
@@ -6,5 +6,6 @@ export const ENVIRONMENT_DEV: Environment = {
 	environment_name: 'development',
 	production: false,
 	valid_chains: { 421613: ARB_GOERLI, 42161: ARB_ONE },
-	default_chain_id: 421613
+	default_chain_id: 421613,
+	dapp_url: 'https://app2.aragog.live',
```
Comment (prateekreddy) [inline src/lib/environments/environment.dev.ts:10]: Why is this necessary ?

Comment (souvikmishra) [reply src/lib/environments/environment.dev.ts:10]: for trezor wallet it is a required parameter, i thought it would be better to keep them configurable on an environment purpose

File: src/lib/controllers/web3OnboardController.ts [hunk 2]
```diff
@@ -30,8 +30,8 @@ const coinbaseWalletSdk = coinbaseWalletModule();
 const gnosis = gnosisModule();
 const infinityWalletSDK = infinityWalletWalletModule();
 const trezor = trezorModule({
-	email: 'roshan@blackfishtech.in',
-	appUrl: 'https://app2.aragog.live'
+	email: 'examplesupport@mail.com',
```
Comment (prateekreddy) [inline src/lib/controllers/web3OnboardController.ts:33]: What are the implications of this email id ?

File: src/lib/environments/environment.dev.ts (hunk 1 above)
Comment (prateekreddy) [reply src/lib/environments/environment.dev.ts:10]: Does Trezor actually use it or just a placeholder ?

Comment (souvikmishra) [reply src/lib/environments/environment.dev.ts:10]: it uses it to maintain a record of dapps using trezor as their wallet itegration

Comment (prateekreddy) [reply src/lib/environments/environment.dev.ts:10]: if we give random link here, will out app still work ?

Comment (souvikmishra) [reply src/lib/environments/environment.dev.ts:10]: yes

File: src/lib/environments/environment.mainnet.ts [hunk 3]
```diff
@@ -5,5 +5,7 @@ export const ENVIRONMENT_MAINNET: Environment = {
 	environment_name: 'mainnet',
 	production: true,
 	valid_chains: { 42161: ARB_ONE },
-	default_chain_id: 42161
+	default_chain_id: 42161,
+	dapp_url: 'https://app.aragog.live',
```
Comment (prateekreddy) [inline src/lib/environments/environment.mainnet.ts:9]: The main net url is arb1.marlin.org ?

**Review by prateekreddy — APPROVED** [review]
